src/pwr.rs

```diff
@@ -308,61 +284,72 @@ impl PowerMode for StopMode<'_> {
         self.scb.set_sleepdeep();
 
         // Restore current clock source after waking up from Stop mode.
-        self.rcc.rb.cfgr.modify(|_, w|
-            match self.rcc.clocks.source() {
+        self.rcc
+            .rb
+            .cfgr
+            .modify(|_, w| match self.rcc.clocks.source() {
                 ClockSrc::MSI(_) =>
-                    // Use MSI as clock source after wake-up
-                    w.stopwuck().clear_bit(),
+                // Use MSI as clock source after wake-up
+                {
```

Comment:
This one looks a bit weird... Since when does rustfmt put opening curly braces on its their own line? Might be a mess-up due to the comment...

Reply:
It probably is. I will rearrange the comment so it does not look that bad. Good catch thanks